code/game/objects/items/ego_weapons/non_abnormality/kcorp.dm

```diff
@@ -0,0 +1,32 @@
+/obj/item/ego_weapon/city/kcorp
```

Comment:
This one is a weirder request, but giving this actual Baton functionality would make this infinitely more interesting.

Reply:
Looked into it, would require a repathing and if you wanted to add Kcorp-wide mechanics it would be significantly harder.